ViennaAdvantage/Areas/VA012/Models/BankStatementModel.cs

```diff
@@ -4251,8 +4379,16 @@ public string CreatePaymentFromOrder(Ctx ctx, List<StatementProp> _formData)
                 //get Bpartner Location
                 int _bPartnerLocation_ID = Util.GetValueOfInt(DB.ExecuteScalar("SELECT C_BPartner_Location_ID FROM C_BPartner_Location WHERE IsActive='Y' AND C_BPartner_ID=" + _formData[0]._ctrlBusinessPartner));
 
+                if (_formData[0]._cmbCurrency != _formData[0]._txtCurrency)
+                {
+                    _txtAmount = MConversionRate.Convert(ctx, _formData[0]._txtAmount, _formData[0]._cmbCurrency, _formData[0]._txtCurrency, _formData[0]._dtStatementDate, _formData[0]._txtConversionType, ctx.GetAD_Client_ID(), _formData[0]._bankAcctOrg_ID);
+                }
+                else {
+                    _txtAmount = _formData[0]._txtAmount;
+                }
+
                 MPayment _pay = new MPayment(ctx, 0, null);
-                int C_Doctype_ID = GetDocTypeID(ctx, _formData[0]._txtAmount);
+                int C_Doctype_ID = GetDocTypeID(ctx, _txtAmount);//Get DocType using Amount
```

Comment:
correct this method, dont pick organization from the context

Reply:
Done.